src/components/TokenOptionItem.tsx

```diff
@@ -177,6 +174,9 @@ export const SearchItem: React.FC<SearchItemProps> = ({ value, defaultText, netw
   }, [value.toLowerCase(), networkId, setFetchResult])
   // don't differentiate based on case
 
+  const { modalProps, addTokensToList } = useBetterAddTokenModal({ focused })
+  console.log('modalProps', modalProps)
```

Comment:
Don't forget to remove this

Reply:
You are checking an old commit, even github flagged this as `outdated`